lib/sieve_cases.rb

```diff
@@ -0,0 +1,33 @@
+class SieveCase < OpenStruct
+  OPEN_ARRAY = "[\n\s\s\s\s\s\s".freeze
+  CLOSE_ARRAY = "\n\s\s\s\s]".freeze
+  NEW_ARRAY_ROW = ",\n\s\s\s\s\s\s".freeze
+  ARRAY_ELEMENTS_PER_ROW = 17.freeze
```

Comment:
In the previous `sieve_test.rb`, the `test_primes_up_to_1000` expected array had 17 elements in its first row, so I took this as the base number. The ideal solution should probably count characters and match with Rubocop rules but I wasn't sure how to do it, so I came up with this solution.